modules/mobile/app/controllers/mobile/v0/clinics_controller.rb

```diff
@@ -16,12 +16,37 @@ def slots
 
         response = systems_service.get_available_slots(location_id: facility_id,
                                                        clinic_id:,
+                                                       clinical_service: nil,
                                                        start_dt: start_date,
                                                        end_dt: end_date)
 
         render json: Mobile::V0::ClinicSlotsSerializer.new(response)
       end
 
+      def facility_slots
+        if !params[:clinic_id] && !params[:clinical_service]
+          render status: :bad_request, json: {
```

Comment:
for readbility, `unless params[:clinic_id] || params[:clinical_service]` to me makes it clearer that you're checking for the absence of both parameters

Reply:
Rubocop didn't like an `unless` with an `else`, and went with `if params[:clinic_id] || params[:clinical_service]`.  Is that better?